packages/apps/shopify-api/lib/billing/types.ts

```diff
@@ -450,6 +450,12 @@ export interface AppPlanDiscount {
   value: AppPlanDiscountAmount;
 }
 
+export interface AppSubscriptionLineItemUpdatePayload {
```

Comment:
We will need to create an additional type like `AppSubscriptionLineItemUpdateResponse` that is just the `confirmationUrl` and the `appSubscription`.

Reply:
Similar to how the cancel [method returns an AppSubscription](https://github.com/Shopify/shopify-app-js/blob/6abecca30bce38f140fbaf31f0137d4a30cf1561/packages/apps/shopify-api/lib/billing/types.ts#L627)

And not a CancelResponse.